README.rst

```diff
@@ -0,0 +1,176 @@
+    ``orquestra.ibmq`` device.  By default tests on the ``orquestra.ibmq``
+    device run with ``ibmq_qasm_simulator`` backend. At the time of writing
+    this means that the test are "free". Please verify that this is also the
+    case for your account.
+
+Further test cases for the ``steps`` used by the PennyLane-Orquestra plugin are
+located in ``steps/tests``. To run these, Python version 3.7 and above is
+required along with the dependencies contained in the
+``steps/requirements_for_tests.txt``. Once these are available, running ``make
+test-steps`` will run the ``steps`` test suite.
+
+.. installation-end-inclusion-marker-do-not-remove
+
+Please refer to the `plugin documentation <https://pennylaneorquestra.readthedocs.io/>`_ as
+well as to the `PennyLane documentation <https://pennylane.readthedocs.io/>`_ for further reference.
+
+Contributing
+============
+
+We welcome contributions - simply fork the repository of this plugin, and then make a
+`pull request <https://help.github.com/articles/about-pull-requests/>`_ containing your contribution.
+All contributers to this plugin will be listed as authors on the releases.
+
+We also encourage bug reports, suggestions for new features and enhancements, and even links to cool projects
+or applications built on PennyLane.
+
+Authors
+=======
+
+PennyLane-Orquestra is the work of `many contributors <https://github.com/PennyLaneAI/pennylane-orquestra/graphs/contributors>`_.
+
+If you are doing research using PennyLane and PennyLane-Orquestra, please cite `our paper <https://arxiv.org/abs/1811.04968>`_:
+
+    Ville Bergholm, Josh Izaac, Maria Schuld, Christian Gogolin, M. Sohaib Alam, Shahnawaz Ahmed,
+    Juan Miguel Arrazola, Carsten Blank, Alain Delgado, Soran Jahangiri, Keri McKiernan, Johannes Jakob Meyer,
+    Zeyue Niu, Antal Száva, and Nathan Killoran.
+    *PennyLane: Automatic differentiation of hybrid quantum-classical computations.* 2018. arXiv:1811.04968
+
+.. support-start-inclusion-marker-do-not-remove
+
+Support
+=======
+
+- **Source Code:** https://github.com/PennyLaneAI/pennylane-orquestra
+- **Issue Tracker:** https://github.com/PennyLaneAI/pennylane-orquestra/issues
+- **PennyLane Forum:** https://discuss.pennylane.ai
+
+If you are having issues, please let us know by posting the issue on our Github issue tracker, or
+by asking a question in the forum.
+
+.. support-end-inclusion-marker-do-not-remove
+.. license-start-inclusion-marker-do-not-remove
+
+License
+=======
+
+The PennyLane orquestra plugin is **free** and **open source**, released under
+the `Apache License, Version 2.0 <https://www.apache.org/licenses/LICENSE-2.0>`_.
+
+.. license-end-inclusion-marker-do-not-remove
+
+A package for integrating PennyLane with Orquestra.
+
+**Installation**
+
+Installing `PennyLane <https://github.com/PennyLaneAI/pennylane>`__ and the `Quantum Engine CLI <https://github.com/zapatacomputing/qe-cli>`__ are required.
```

Comment:
The `installing` and `are required` reads a bit weird. Perhaps rewrite this slightly?

```suggestion
`PennyLane <https://github.com/PennyLaneAI/pennylane>`__ and the `Quantum Engine CLI <https://github.com/zapatacomputing/qe-cli>`__ are required.
```

Reply:
Oh, that's a fair point, thanks! Actually this sections seems to be a duplicate, so removing these (already have an `Installation` section).